clients/extension/src/utils/constants.ts

```diff
@@ -265,7 +265,7 @@ export const explorerUrl: ChainStrRef = {
   [ChainKey.OSMOSIS]: "https://www.mintscan.io/osmosis",
   [ChainKey.POLKADOT]: "https://polkadot.subscan.io/account",
   [ChainKey.POLYGON]: "https://polygonscan.com",
-  [ChainKey.SOLANA]: "https://explorer.solana.com",
```

Comment:
@Ehsan-saradar we have the same definition in `clients/desktop/chain/utils/getBlockExplorerUrl` could you please see whether you can reuse those definition , so we only define it once